apps/site/lib/site_web/plugs/canonical_hostname.ex

```diff
@@ -32,8 +34,18 @@ defmodule SiteWeb.Plugs.CanonicalHostname do
       |> put_status(:moved_permanently)
       |> redirect(external: rewritten_url)
       |> halt()
-    else
-      conn
+    end
+  end
+
+  # The health checker uses a local IP hostname
+  defp is_local_ip(hostname) do
```

Comment:
Will this approach work ok in both prod and dev, and are the addresses something that should be specified in the config?

Reply:
It should work on both, I just double-checked Splunk and the `/_health` endpoint is being hit from a local IP there as well, so I assume that it's requesting the local IP as the host.

The specific addresses it uses seem arbitrary so I don't want to specify them, rather just to check that it's some private IP or another.